cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -228,9 +227,17 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
           val method = frame.getMethod
           if(method ne null) {
             val splitIndex = method.getType.getName.lastIndexOf(".")
+            var packageName:String = ""
+            var methodName:String = ""
+            if(splitIndex != -1) {
+              packageName = method.getType.getName.substring(0, splitIndex).intern()
+              methodName = method.getType.getName.substring(splitIndex + 1).intern()
+            }else {
+              methodName = method.getType.getName
+            }
             CallSite(
-              method.getType.getName.substring(0,splitIndex).intern(),
-              method.getType.getName.substring(splitIndex+1).intern(),
+              packageName.intern(),
```

Comment:
why are you interning

Reply:
We don't want different copies of that particular string to exist. Allow of callSite could have the same packageName, className but different line numbers

Reply:
look at your code, and work out what you need to intern, and where

Reply:
found it.